src/KamadaKawai.jl

```diff
@@ -0,0 +1,87 @@
+import Optim
+using Distances
+function kamada_kawai_layout(G, X=nothing; C= 1.0, MAXITER=100 )
```

Comment:
I think we need some kind of docstring here, explaining what happens.

Also we should use lower capitel letters for variable names.

Reply:
Not a problem. I've never done a non-trivial pull-request, so bear with me as I work my way through the process.

Reply:
I see that `stressmajorize_layout` using upper case variable names. I personally prefer CamelCase for Variables and camelCase for functions.

Reply:
You probably mean `MAXITER` more so than `X` or `G`. Done.